Tasks/DeployVisualStudioTestAgent/SetupTestMachineForUITests.ps1

```diff
@@ -42,16 +42,17 @@ function isShowLogonMessagePolicyEnabled([int] $OSType)
 	}
 	
 	$legalCaption = (Get-ItemProperty $registryPath -ErrorAction SilentlyContinue).legalnoticecaption
-	if (($legalCaption) -and ($legalCaption.Length -gt 0))
+	$legalText = (Get-ItemProperty $registryPath -ErrorAction SilentlyContinue).legalnoticetext
+	if (($legalCaption) -and ($legalCaption.Length -gt 0) -and ($legalText) -and ($legalText.Length -gt 0))
```

Comment:
refactor this code. IsShowLogonMessagePolicyEnabled and IsShowLogonMessageEnabled are essentially doing the same